installer_scripts/install-yb-voyager

```diff
@@ -173,6 +183,36 @@ macos_main() {
 # COMMON
 #=============================================================================
 
+check_java() {
+		if [ -z "$JAVA_HOME" ]; then
+			JAVA_BINARY="java"
+		else
+			JAVA_BINARY="$JAVA_HOME/bin/java"
+		fi
+
+        MIN_REQUIRED_MAJOR_VERSION='11'
+		JAVA_MAJOR_VER=$(${JAVA_BINARY} -version 2>&1 | awk -F '"' '/version/ {print $2}' | awk -F. '{print $1}')
+		
+        if ([ -n "$JAVA_MAJOR_VER" ] && (( 10#${JAVA_MAJOR_VER} >= 10#${MIN_REQUIRED_MAJOR_VERSION} )) ) #integer compare of versions.
+        then
+        	output "Found sufficient java version = ${JAVA_MAJOR_VER}"
+        else
+        	output "ERROR: Java not found or insuffiencient version ${JAVA_MAJOR_VER}. Please install java>=${MIN_REQUIRED_MAJOR_VERSION}"
+			exit 1;
+        fi
+}
```

Comment:
@amit-yb do you still want the java version check to be simplified to just a binary present or not check?

Reply:
Done